core/layout_api.php

```diff
@@ -397,14 +397,14 @@ function layout_navbar() {
 
 	# mobile view
 	echo '<div class="hidden-sm hidden-md hidden-lg">';
-	echo '<nav class="navbar-menu pull-left navbar-collapse collapse" role="navigation" style="height: auto;">';
```

Comment:
I don't think we need the <nav> element anymore. It used to be part of the UI theme in early version but does not seem to be the case anymore.

http://responsiweb.com/themes/preview/ace/1.4/index.html